numpy/lib/_iotools.py

```diff
@@ -122,7 +122,7 @@ def flatten_dtype(ndtype, flatten_base=False):
     ----------
     ndtype : dtype
         The datatype to collapse
-    flatten_base : {False, True}, optional
+    flatten_base : bool, default false
```

Comment:
the standard way to write this is ``bool, optional``. And in the parameter description "default is False".